legal_db/static/webpack_files/css/index.css

```diff
@@ -1,2 +1,2 @@
bottom:.25rem !important}.padding--small{padding:.5rem !important}.padding-top-small,.sidebar-menu .menu-list .link{padding-top:.5rem !important}.padding-right-small,.table-of-contents .icon,.sidebar-menu .icon{padding-right:.5rem !important}.padding-bottom-small{padding-bottom:.5rem !important}.padding-left-small{padding-left:.5rem !important}.padding-horizontal-small,.image.is-compact .image-icons,.image .bookmark-icon,.card.entry-post.project-index .card-content .external-links .button .link-content,.card.entry-post.project-index .card-content .site-link .icon{padding-left:.5rem !important;padding-right:.5rem !important}.padding-vertical-small,.image .image-title,.image .image-icons,.image .bookmark-icon,.tabs.is-boxed a{padding-top:.5rem !important;padding-bottom:.5rem !important}.padding--normal{padding:1rem !important}.padding-top-normal,.sidebar-menu{padding-top:1rem !important}.padding-right-normal{padding-right:1rem !important}.padding-bottom-normal{padding-bottom:1rem !important}.padding-left-normal{padding-left:1rem !important}.padding-horizontal-normal,.image .image-icons,.image .image-title,.tabs.is-boxed a{padding-left:1rem !important;padding-right:1rem !important}.padding-vertical-normal,.navbar.is-compact,.notification.is-compact .notification-container{padding-top:1rem !important;padding-bottom:1rem !important}.padding--big{padding:1.5rem !important}.padding-top-big{padding-top:1.5rem !important}.padding-right-big{padding-right:1.5rem !important}.padding-bottom-big,.sidebar-menu{padding-bottom:1.5rem !important}.padding-left-big{padding-left:1.5rem !important}.padding-horizontal-big,.sidebar-menu .menu-list .link{padding-left:1.5rem !important;padding-right:1.5rem !important}.padding-vertical-big{padding-top:1.5rem !important;padding-bottom:1.5rem !important}.padding--bigger{padding:2rem !important}.padding-top-bigger{padding-top:2rem !important}.padding-right-bigger{padding-right:2rem !important}.padding-bottom-bigger{padding-bottom:2rem !important}.padding-left-bigger{padding-left:2rem !important}.padding-horizontal-bigger,.navbar.is-compact{padding-left:2rem !important;padding-right:2rem !important}.padding-vertical-bigger{padding-top:2rem !important;padding-bottom:2rem !important}.padding--large{padding:2.5rem !important}.padding-top-large{padding-top:2.5rem !important}.padding-right-large{padding-right:2.5rem !important}.padding-bottom-large{padding-bottom:2.5rem !important}.padding-left-large{padding-left:2.5rem !important}.padding-horizontal-large{padding-left:2.5rem !important;padding-right:2.5rem !important}.padding-vertical-large{padding-top:2.5rem !important;padding-bottom:2.5rem !important}.padding--larger{padding:3rem !important}.padding-top-larger{padding-top:3rem !important}.padding-right-larger{padding-right:3rem !important}.padding-bottom-larger{padding-bottom:3rem !important}.padding-left-larger{padding-left:3rem !important}.padding-horizontal-larger{padding-left:3rem !important;padding-right:3rem !important}.pa
```

Comment:
I have gone through the sass file but as we can see the CSS file is way more longer then the SASS file and the SASS file is just 295 lines long and as in the CSS file, in the code the anchor tag with class = "navbar-burger"  is not even given any style, there is no use ".navbar-burger" anywhere and I am not able to understand how is this even possible.
Also I have looked through the whole code but there was only one SASS file.
Can you please tell me if I am doing anything wrong or is there anything that I am not able to find.
@TimidRobot

Reply:
Okay,
But what can I do resolve problem of multiple files being involved in generating the css file?
and where can I find the style given to the anchor tag with class="navbar-burger", so that I can update the style given o it and resolve this issue?

@TimidRobot

Reply:
Currently, I am not familiar with SASS and I am learning it so for now I am closing this PR.